docs/models.rst

```diff
@@ -65,20 +65,35 @@ Every model should be derived from base model. You also can derive from your own
 
 This models won't be created in schema generation and won't create relations to other models.
 
-Further, let's take a look at created fields for models
 
-.. code-block:: python3
+Further we have field ``fields.DatetimeField(auto_now=True)``. Options ``auto_now`` and ``auto_now_add`` work like Django's options.
 
-    id = fields.IntField(pk=True)
+Primary Keys
+------------
 
-This code defines integer primary key for table.
+In Tortoise ORM we require that a model has a primary key.
 
-Currently we **only** support the primary key to be called ``id``, and to be an ``IntField``.
+That primary key will be accesible through a reserved field ``pk`` which will be an alias of whichever field has been nominated as a primary key.
+We currently support single (non-composite) primary keys of any indexable field type, but only these field types are recommended:
+
+* ``IntField``
+* ``BigIntField``
+* ``CharField``
+* ``UUIDField``
+
+One must define a primary key by setting a ``pk`` parameter to ``True``.
 If you don't define a primary key, we will create a primary key of type ``IntField`` with name of ``id`` for you.
 
-Sadly, currently only simple integer primary key is supported, there is plans to enhance this in the not too distant future.
+Any of these are valid primary key definitions in a Model:
+
+.. code-block:: python3
+
+    id = fields.IntField(pk=True)
+
+    checksum = fields.CharField(pk=True)
+
+    guid = fields.UUIDField(pk=True)
```

Comment:
I think we can also mention, that filtering by `.filter(pk__in=...)` or whatever other filters is also supported